package model

class Archive(
    val title: String,
) {
    val notes: MutableList<Note> = mutableListOf()
}